packages/data/tests/dummy/mirage/scenarios/graphql-blockhead.ts

```diff
@@ -1,11 +1,25 @@
 // TODO: Replace any with type supplied by new version of mirage
+import { grains } from './graphql';
```

Comment:
Can we rebase on this with the new file name?

Reply:
The new file names are in the request v2 branch and this is being merged into master